code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1033,6 +1033,17 @@
 /mob/living/carbon/human/updatehealth()
 	. = ..()
 	dna?.species.spec_updatehealth(src)
+	if(HAS_TRAIT(src, TRAIT_IGNOREDAMAGESLOWDOWN))
+		remove_movespeed_modifier(MOVESPEED_ID_DAMAGE_SLOWDOWN)

```

Comment:
What about in here ? should that trait also stop flyers from slowing down ?

Reply:
yeah, probably should have caught that myself